src/Mechanical/Rotational/components.jl

```diff
@@ -167,7 +167,7 @@ Linear 1D rotational spring and damper
     @parameters begin
         d, [description = "Damping constant"]
         c, [description = "Spring constant"]
-        phi_rel0 = 0.0, [description = "Unstretched spring angle"]
+        phi_rel0, [description = "Unstretched spring angle", guess = 0.0]
```

Comment:
This is an oversight; will revert.

Reply:
Resolved here: https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/274/commits/eae592ba86e9de11fa575f005d3c13b526a80c99#diff-36a0d1920684771c0166099143e51f1cf58f1faae7d847cb151c0a0826c4348bR170